<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageUsers</class>
 <widget class="QDialog" name="ManageUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户管理</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTableWidget" name="usersTable">
     <property name="minimumSize">
      <size>
       <width>541</width>
       <height>261</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>11</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>12</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>13</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>15</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>学号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>密码</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>QQ邮箱</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>注册时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>在线状态</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QTableWidget" name="borrowTableWidget">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>300</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>11</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>12</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>13</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>14</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>15</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>书号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>借阅时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>到期时间</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="remindreturnButton">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>提醒归还</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="refreshButton">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="closeButton">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>usersTable</zorder>
  <zorder>borrowTableWidget</zorder>
  <zorder>remindreturnButton</zorder>
  <zorder>closeButton</zorder>
  <zorder>refreshButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
